guides/service design at dxw

```diff
@@ -0,0 +1,42 @@
+# Service Design approach
```

Comment:
This file needs to be renamed to something like `service-deign-at-dxw.md`. The file extension is important.